### PR TITLE
Update typeahead script and use ip addresses when searching hosts

### PR DESCRIPTION
**Pull request by emanuele-f**
The migration of existing code to avoid hostname-based access to the host_details page should be gradual since it is spread across different scripts in the codebase. This patch only fixes search bars related to #835.

### CODE REVIEW

File: scripts/lua/find_host.lua [hunk 1]
```diff
@@ -25,10 +25,30 @@ print [[
...
+	          values[v] = values[v] + 1
+	       end
+	    end
+	 end
+	 
+	 for k, v in pairs(res) do
+	    if(v ~= "") then
+	       if values[v] > 1 then
+	          -- we matched both an ipv4 and ipv6 with same host name, display differently
+	          if findString(v, ":") == nil then
```
Comment (simonemainardi) [inline scripts/lua/find_host.lua]: check lua_utils there are already functions that check if the IP is v4 or v6 and they should also be more accurate

**Review by simonemainardi — CHANGES_REQUESTED** [review]

File: scripts/lua/find_host.lua [hunk 2]
```diff
@@ -35,15 +35,15 @@ print [[
 	       end
 	    end
 	 end
-	 
+	 tprint(res)
```
Comment (simonemainardi) [inline scripts/lua/find_host.lua]: remove this debug line

**Review by simonemainardi — APPROVED** [review]

**Comment by simonemainardi** [comment]
thank you @emanuele-f

**Comment by lucaderi** [comment]
@emanuele-f @simonemainardi This code is bugged and never tested. See " if isIPv6String(v) then" in find_host.lua and you will understand why it doesn't work.

**Pull request merged**